examples/strategies/turtle.go

```diff
@@ -0,0 +1,66 @@
+package strategies
+
+import (
+	"github.com/rodrigo-brito/ninjabot"
+	"github.com/rodrigo-brito/ninjabot/service"
+
+	"github.com/markcheno/go-talib"
+	log "github.com/sirupsen/logrus"
+)
+
+// https://www.investopedia.com/articles/trading/08/turtle-trading.asp
+type Turtle struct{}
+
+func (e Turtle) Timeframe() string {
+	return "4h"
+}
+
+func (e Turtle) WarmupPeriod() int {
+	return 40
+}
+
+func (e Turtle) Indicators(df *ninjabot.Dataframe) {
+	df.Metadata["turtleHighest"] = talib.Max(df.Close, 40)
+	df.Metadata["turtleLowest"] = talib.Min(df.Close, 20)
+}
+
+func (e *Turtle) OnCandle(df *ninjabot.Dataframe, broker service.Broker) {
+	closePrice := df.Close.Last(0)
+	maxS := df.Metadata["turtleHighest"]
+	highest := maxS[len(maxS)-1]
+	minS := df.Metadata["turtleLowest"]
+	lowest := minS[len(minS)-1]
+	assetPosition, quotePosition, err := broker.Position(df.Pair)
+	if err != nil {
+		log.Error(err)
```

Comment:
I think a return will be great here. Avoid wrong operations.
```suggestion
		log.Error(err)
		return
```

Reply:
It should be done also in ocosell and emacross. They have same issue.

Reply:
Yes, can you fix it in other strategies?